src/Stratis.Bitcoin.Features.BlockStore/ControllersAndClients/BlockStoreClient.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using NBitcoin;
+using Stratis.Bitcoin.Controllers;
+
+namespace Stratis.Bitcoin.Features.BlockStore.ControllersAndClients
```

Comment:
Perhaps this namespace should be just Clients?

Reply:
It contains also block store controller

I think it makes sense to keep controllers and clients in the same folder since clients just reflect the corresponding controllers

Reply:
@noescape00 I'm going to have to fight you on this :P Traditionally, controller classes are always in their own folders. HttpClients is separate functionality and should be in their own folder as well.

I had a look at your branch in VS and the client methods are not called from the controllers?

Reply:
>I had a look at your branch in VS and the client methods are not called from the controllers?

No. Client calls the methods from controllers but not directly.

>Traditionally, controller classes are always in their own folders. 

It just feels weird to have a folder for a single controller. And since client and controller reflect each other I see nothing bad in keeping them in a single folder

Reply:
OK if you don't want to create a new folder for the client then just call the folder `Controllers`. `Stratis.Bitcoin.Controllers` has this convention, so let's keep it the same then?

Reply:
Ok, sure